### DISCUSSION TITLE
Release v1.6.8

### DISCUSSION THREAD

**Discussion by chinosk6** [chinosk6/gakuen-imas-localify#49] [Announcements]
# v1.6.8
- Fixed crashes on some devices when using remote files.
- Adapted for game version `1.7.0`.

---

# v1.6.8
 - 修复部分机型使用远程文件崩溃的问题
 - 适配游戏 `1.7.0`

<hr /><em>This discussion was created from the release <a href='https://github.com/chinosk6/gakuen-imas-localify/releases/tag/v1.6.8'>Release v1.6.8</a>.</em>

**Comment by tatsuya087** [comment]
When I try to start gakumas localify in LDPlayer, it crashes, any solution?
[https://youtu.be/mR5Ra6WxN1c](https://youtu.be/mR5Ra6WxN1c)

**Reply by rizkidewanto** [reply]
Same to me

**Reply by chinosk6** [reply]
Maybe the Android version is too low. Try the MuMu Player.
